### ISSUE TITLE
JS should not add 'aria-expanded=false' to .mzp-c-navigation-items

### ISSUE THREAD

**Issue by stephaniehobson** [mozilla/protocol#847]
`aria-expanded=false` is being added to `.mzp-c-navigation-items`. 

This role should be declared on a controlling element like a button or link and must be used with `aria-controls`. Also, on desktop this content is expanded by default so it should not be false.

Wild speculation about the cause of this:

I think it should be on `c-navigation-menu-button` instead. Since the button is hidden I don't know if it would be confusing to a screen reader to have it declared false or not. We can fix it to be true on desktop or do research to see if that's necessary and document that it is not.

**Comment by nathan-barrett** [comment]
Can this be closed? I think it was fixed in https://github.com/mozilla/protocol/pull/860

**Comment by stephaniehobson** [comment]
Fixed in #860
